### ISSUE TITLE
Implement homepage design

### ISSUE THREAD

**Issue by cthibaultatx** [cityofaustin/janis#33]
- [x] update tile/service and module/services in sm, med, and lge

- [x] add 'work in progress' message in sm, med, and lge

- [ ] ~add theme to main navigation plus...~

- [ ] ~...refine mobile and tablet menu and develop new large/desktop menu~

- [ ] ~Themes and topics are listed here: [Future Navigation airtable](https://airtable.com/shr3RatfCvSZmJhqw) Only build in topics we already have services/content for ('Recycling, Trash, and Compost' under 'Housing & Utilities'). Leave off other topics and instead, when a user hovers over a theme with no topics- they should read: (waiting on copy from content team @courtneyjacinic )~

- [ ] ~add airport icon to tablet and desktop header, and in mobile menu- links to http://austintexas.gov/department/airport~

- [ ] ~add 311 link to header to tablet and desktop, and in mobile menu- links to http://311.austintexas.gov/reports/list_services~

- [x] ~add 'alpha' to 'austin.gov' = 'alpha.austin.gov'~

## Do not change these things:
- language header
- header/footer colors
- 311 and footer content - will be separate issue
- typestyles

![alpha_l_0 0_home_menua](https://user-images.githubusercontent.com/20911665/36178991-5141132a-10e0-11e8-9f7b-ac40a9b02b17.png)

![alpha_l_0 0_home_menub](https://user-images.githubusercontent.com/20911665/36178992-5154bc7c-10e0-11e8-9a3a-76c74105410b.png)

![alpha_l_0 0_home](https://user-images.githubusercontent.com/20911665/36178993-516a44a2-10e0-11e8-9815-1b68c8160c32.png)

![alpha_m_0 0_home_menu](https://user-images.githubusercontent.com/20911665/36178994-517d4d22-10e0-11e8-8d3a-97254cbc42a5.png)

![alpha_m_0 0_home](https://user-images.githubusercontent.com/20911665/36178995-5190cf50-10e0-11e8-8e63-cec6f5f84626.png)

![alpha_s_0 0_home_menu](https://user-images.githubusercontent.com/20911665/36178996-51a5d832-10e0-11e8-9fae-36cb09da41eb.png)

![alpha_s_0 0_home](https://user-images.githubusercontent.com/20911665/36178997-51bc0166-10e0-11e8-901e-ed456f060121.png)

image for alpha.austin.gov: https://www.flickr.com/photos/austintexasgov/13248971115/in/album-72157642552791885/ you can download the right size from the flickr source image

## Resident Value
Creating a warm and respectful welcome message to residents and prioritizing their primary paths to getting what they need

**Comment by mateoclarke** [comment]
@cthibaultatx, I know you've been doing more work recently iterating on the homepage design. Will you let us know when it's in a good place to continue work? I think we can knock it out this week.

**Comment by mateoclarke** [comment]
I moved the checklist items in this issue's description relating to updates to the global navigation into issue #124.

**Comment by mateoclarke** [comment]
@amenity, In the description above, Celine said that the 311 and footer updates will be addressed in a separate issue. Since your comment about blue on blue color contrast relates to that section, will you make sure that feedback gets added somewhere like https://github.com/cityofaustin/techstack/issues/102 so it doesn't get lost?
